test/__manifest__.py

```diff
@@ -1,16 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo and Trescloud. See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'Test',
-    'version': '1.0',
-    'category': 'Test',
-    'sequence': 15,
-    'summary': 'Test Module',
-    'website': 'trescloud.com',
-    'depends': [
-    ],
-    'data': [
-    ],
-    'installable': True,
-    'application': True,
-    'auto_install': True,
-    'license': 'LGPL-3',
+	'name': 'Test Jerry',
+	'version': '1.0',
+	'category': '',
+	'description': '''
+ 		Personalizaciones 
+	Autores:
+	- Jerry Rivera
+	''',
+	'author': 'TRESCLOUD',
+	'maintainer': 'TRESCLOUD',
+	'website': 'https://www.trescloud.com',
+	'summary': '',
+	'license': 'OEEL-1',
+	'depends': [
+        'l10n_ec_edi',
```

Comment:
```suggestion
        'l10n_ec_edi',
```
este modulo no es necesario. basta con el account